### DISCUSSION TITLE
How to use MYRIAD plugin with OpenVINO Notebooks

### DISCUSSION THREAD

**Discussion by raymondlo84** [openvinotoolkit/openvino_notebooks#639] [General]
https://github.com/openvinotoolkit/openvino/issues/11476

So since 2022.1, the plugin is no longer included in the Pypi...

The easiest way to get around it is to follow the instruction above.

In short, it will be
1. Install OpenVINO with offline tool
2. Run the command with setupvar.sh
3. Activate the virtual environment (openvino_env) and run jupyter lab notebook command as usual.

:)

**Comment by adrianboguszewski** [comment]
For Windows: the second step is not required - it should work out of the box
For Linux: run `setupvar.sh` or look at [this](https://medium.com/openvino-toolkit/how-to-run-openvino-with-neural-compute-stick-2-on-linux-9ab1f185c920)
